en/views/helpers/form.rst

```diff
@@ -187,7 +187,7 @@ rules that only apply when an account is being registered::
         'context' => ['validator' => 'register']
     ]);
 
-The above will use the ``register`` validator for the ``$user`` and all related
+The above will use the ``register`` validator(the rules defined within UsersTable::validationRegister()) for the ``$user`` and all related
```

Comment:
`The above will use the ``register`` validator, the rules defined within UsersTable::validationRegister(), for`..